Sources/Support/PurchasesDiagnostics.swift

```diff
@@ -167,11 +263,41 @@ extension PurchasesDiagnostics.Error: CustomNSError {
 
     var localizedDescription: String {
         switch self {
+        case .notAuthorizedToMakePayments: return "The person is not authorized to make payments on this device."
         case let .unknown(error): return "Unknown error: \(error.localizedDescription)"
         case let .failedConnectingToAPI(error): return "Error connecting to API: \(error.localizedDescription)"
         case let .failedFetchingOfferings(error): return "Failed fetching offerings: \(error.localizedDescription)"
         case let .failedMakingSignedRequest(error): return "Failed making signed request: \(error.localizedDescription)"
         case .invalidAPIKey: return "API key is not valid"
+        case .invalidSDKVersion: return "SDK Version is not supported"
+        case .noOfferings: return "No offerings configured"
+        case let .offeringConfiguration(payload):
+            guard let offendingOffering = payload.first(where: { $0.status == .failed }) else {
+                return "Default offering is not configured correctly"
+            }
+
+            let offeringIdentifier = offendingOffering.identifier
+            let offendingPackageCount = offendingOffering.packages.filter({ $0.status != .valid }).count
+            let noPackages = "Offering '\(offeringIdentifier)' has no packages"
+            let packagesNotReady = """
+            Offering '\(offeringIdentifier)' uses \(offendingPackageCount) products \
+            that are not ready in App Store Connect.
+            """
+
+            return offendingOffering.packages.isEmpty ? noPackages : packagesNotReady
+        case let .invalidBundleId(payload):
+            guard let payload else {
+                return "Bundle ID in your app does not match the Bundle ID in the RevenueCat dashboard"
+            }
+            let sdkBundleId = payload.sdkBundleId
+            let appBundleId = payload.appBundleId
+            return "Bundle ID in your app '\(sdkBundleId)' does not match the RevenueCat app Bundle ID '\(appBundleId)'"
```

Comment:
I'm confused about these two, which one is which? 
does "app" here represent the configuration in the web app?

Reply:
would be killer to add a link to the app configuration in RC web so that we can put it in these and then you can easily go fix web side

Reply:
Yeah tbf I did struggle with the wording of this and reading it back it is incredibly confusing hahaha.

Also, that is a killer feature!! Let me tweak the BE and add a small button for it in the debug view model, be right back hahaha  👀

Reply:
Added it!!

![RocketSim_Screenshot_iPhone_16_Pro_6 3_2025-04-11_14 14 16](https://github.com/user-attachments/assets/9d031aa4-b657-4fe5-a750-75ea8f4ce5ee)

Reply:
😻😻😻